frontend/src/components/dialogs/SecretDialogOpenstack.vue

```diff
@@ -209,7 +216,18 @@ export default {
       applicationCredentialName: undefined,
       applicationCredentialSecret: undefined,
       hideApplicationCredentialSecret: true,
-      validationErrors
+      validationErrors,
+      authenticationMethodValue: 'user',
```

Comment:
- wasn't the naming scheme for such properties *Internal instead of *Value?
- change default to `appCredentials`?